h/presenters/annotation_json.py

```diff
@@ -9,68 +9,59 @@
 
 
 class AnnotationJSONPresenter(AnnotationBasePresenter):
-
     """Present an annotation in the JSON format returned by API requests."""
 
     def __init__(self, annotation_context, formatters=None):
         super().__init__(annotation_context)
 
-        self._formatters = []
-
-        if formatters is not None:
-            self._formatters.extend(formatters)
+        self._formatters = tuple(formatters or [])
 
     def asdict(self):
-        docpresenter = DocumentJSONPresenter(self.annotation.document)
-
-        base = {
-            "id": self.annotation.id,
-            "created": self.created,
-            "updated": self.updated,
-            "user": self.annotation.userid,
-            "uri": self.annotation.target_uri,
-            "text": self.text,
-            "tags": self.tags,
-            "group": self.annotation.groupid,
-            "permissions": self.permissions,
-            "target": self.target,
-            "document": docpresenter.asdict(),
-            "links": self.links,
-        }
+        annotation = deepcopy(self.annotation.extra) or {}
```

Comment:
I guess the `or` is because `annotation.extra` might be `None` in which case `deepcopy()` would return `None`? I'm not sure whether it's necessary because `models.Annotation.extra` is not nullable. The code was already like this though